config/install/field.storage.node.field_people.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_people
+field_name: field_people
+entity_type: node
+type: entity_reference
+settings:
+  target_type: node
+module: core
+locked: false
+cardinality: 1
```

Comment:
This field should accept unlimited values.
![species_field_people_cardinality](https://github.com/notS0Solo/swapimigration/assets/1434979/39cded47-7ec5-48b8-a94e-6e5e68307e16)

Reply:
Fixed